modules/actions/config.go

```diff
@@ -19,6 +19,14 @@ func NewConfig(port uint, remoteDetails *remote.Details) *Config {
 	}
 }
 
+// DefaultConfig returns the default configuration
+func DefaultConfig() Config {
+	return Config{
+		Port: 3000,
+		Node: nil,
```

Comment:
```suggestion 
		Node: remote.DefaultDetails(),
```
We can pass default node config here

Reply:
I've actually intentionally left it `nil` there because I think that the default behavior should be to use the same node that other data is parsed from when handling actions. Using a custom node for actions should be something optional and not required. Do you think it's better to force the users to specify another node there by default instead?

Reply:
Alright let's keep it this way and I will update the docs to explain how to setup custom node address for hasura actions